src/common/options.cc

```diff
@@ -1100,6 +1100,10 @@ std::vector<Option> get_global_options() {
     .set_default(96)
     .set_description(""),
 
+    Option("ms_accept_error_nums", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(4)
+    .set_description("error numbers before do ceph_abort() "),
```

Comment:
could you be more specific on what kind of "error" is in this context? also i am afraid that `ceph_abort()` is not a user-facing term.